desktop-exporter/app/components/detail-view/fields-pannel.tsx

```diff
@@ -29,8 +29,72 @@ export function FieldsPannel(props: FieldsPannelProps) {
       </TabPanel>
     );
   }
+
+  // Root span: label with a little tag
+  let isRoot = span.parentSpanID.length ? false : true;
+  let rootTag = isRoot ? (
+    <Tag
+      marginStart={2}
+      colorScheme="cyan"
+      variant="subtle"
+    >
+      root
+    </Tag>
+  ) : null;
+
+  // parentSpanID: omit field if the span is root
+  let parentSpanIDField = !isRoot ? (
+    <SpanField
+      fieldName="parent span id"
+      fieldValue={span.parentSpanID}
+    />
+  ) : null;
+
+  // Duration: label in appropriate human-readable time unit (s, ms, μs, ns)
   let durationString = getDurationString(span.startTime, span.endTime);
 
+  // Dropped Counts: omit if they are equal to 0
+  let droppedAttributesCountField =
+    span.droppedAttributesCount > 0 ? (
+      <SpanField
+        fieldName="dropped attributes count"
+        fieldValue={span.droppedAttributesCount}
+      />
+    ) : null;
+
+  let droppedEventsCountField =
+    span.droppedEventsCount > 0 ? (
+      <SpanField
+        fieldName="dropped events count"
+        fieldValue={span.droppedEventsCount}
+      />
+    ) : null;
+
+  let droppedLinksCountField =
+    span.droppedLinksCount > 0 ? (
+      <SpanField
+        fieldName="dropped links count"
+        fieldValue={span.droppedLinksCount}
+      />
+    ) : null;
+
+  let droppedResourceAttributesCountField =
+    span.resource.droppedAttributesCount > 0 ? (
+      <SpanField
+        fieldName="dropped attributes count"
+        fieldValue={span.resource.droppedAttributesCount}
+      />
+    ) : null;
+
+  let droppedScopeAttributesCountField =
+    span.scope.droppedAttributesCount > 0 ? (
+      <SpanField
+        fieldName="dropped attributes count"
+        fieldValue={span.scope.droppedAttributesCount}
+      />
+    ) : null;
```

Comment:
Another potential approach here might be to add a `hidden` prop to `SpanField` so you could do something like this inline:

```jsx
      <SpanField
        fieldName="dropped attributes count"
        fieldValue={span.scope.droppedAttributesCount}
        hidden={span.scope.droppedAttributesCount === 0}
      />
```

where `SpanField` returns `null` if `hidden === true`

Reply:
Oooooh. Yes! Good...
![raccoon-evil-smile](https://user-images.githubusercontent.com/56372758/207223770-fdb22512-1a21-4be9-96d3-09d684f1f495.gif)